priv/repo/migrations/20170317153551_create_message.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Healthlocker.Repo.Migrations.CreateMessage do
+  use Ecto.Migration
+
+  def change do
+    create table(:messages) do
+      add :body, :string, null: false
+      add :name, :string
```

Comment:
I guess the main reason was for displaying old messages. I was thinking there would have to be a lot more calls to the db to get the name as well if it wasn't stored in that table (as in a call for each message to get the user from user_id and then get the name). Unless there's a quicker way to get data from an association? Or maybe that wouldn't be as slow as I'm thinking it would be.

Right now it's done in the `join` function like this:

```
def join("room:general", message, socket) do
    ...
    messages = Repo.all(Message)
    resp = %{messages: Phoenix.View.render_many(messages, MessageView, "message.json")}
    {:ok, resp, socket}
  end
```